pkgs/servers/nosql/redis/default.nix

```diff
@@ -18,14 +20,17 @@ stdenv.mkDerivation rec {
     ''}
   '';
 
-  buildInputs = [ lua pkgconfig ] ++ stdenv.lib.optional (stdenv.isLinux && !stdenv.hostPlatform.isMusl) systemd;
+  buildInputs = [ lua pkgconfig ]
```

Comment:
Oh looks like `pkg-config`  in the wrong spot here. It belongs in `nativeBuildInputs`. Do you mind fixing that up while you're in here?

Reply:
Done!